src/solvers/refinement/string_constraint_generator_indexof.cpp

```diff
@@ -373,7 +301,7 @@ exprt string_constraint_generatort::add_axioms_for_last_index_of(
       equal_exprt(str[index], c)));
   axioms.push_back(a3);
 
-  symbol_exprt n=fresh_univ_index("QA_last_index_of", index_type);
+  symbol_exprt n=fresh_univ_index("QA_last_index_of1", index_type);
```

Comment:
The final suffix is a number which is incremented globally for any symbol. I think it makes sense to put our own number here to figure out which axiom variable is being used and will not be confusing as there is a `#` separator.